### ISSUE TITLE
[Suggestion] Option to disable lying down

### ISSUE THREAD

**Issue by wavian** [MehVahdJukaar/sleep_tight#29]
I was messing around with the Fabric version and I noticed you weren't able to disable lying down in general. I know you can disable it if you are on cooldown, but it would be nice to have to option to remove the mechanic.

**Comment by MehVahdJukaar** [comment]
that would almost completely invalidate hammocks wont it? Does that make sense?

**Comment by Rebel459** [comment]
Lying down seems to lead to a lot of desync issues with other mods, where the player is rejected from the bed on the client and the server gets confused, not knowing, meaning the player can effectively free-roam with invincibility and the game acts as if they’re in their bed and behaves as such. Really buggy weirdness

**Comment by MehVahdJukaar** [comment]
What other mods? If it causes issues then they are likely to be altring sleeping mechanic itself so it makes sense it's not compatible

**Comment by Rebel459** [comment]
Nah, none of them alter the mechanics of sleeping. Idk which mod it was, if i get the time I may test to work that out in the future

**Comment by B1gBadClown** [comment]
> that would almost completely invalidate hammocks wont it? Does that make sense?

For me, the hammock's main purpose is to let the player sleep through the day. The ability to lay on it and swing around has always felt more like a role-playing bonus in my opinion. I'm not sure how hard it would be to implement, but perhaps doing something similar to the "#Allows laying on a bed when you are on sleeping cooldown" config option but for hammock would be satisfactory?

**Comment by MehVahdJukaar** [comment]
There's already a config that commits sleeping immediately
